src/CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/AzureQueueStorageTransportBindingElement.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Net;
+using CoreWCF.Configuration;
+using CoreWCF.Queue.Common;
+using CoreWCF.Queue.Common.Configuration;
+using Azure.Storage.Queues;
+
+namespace CoreWCF.Channels
+{
+    public class AzureQueueStorageTransportBindingElement : QueueBaseTransportBindingElement
+    {
+        /// <summary>
+        /// Creates a new instance of the AzureQueueStorageTransportBindingElement Class using the default protocol.
+        /// </summary>
+        public AzureQueueStorageTransportBindingElement()
+        {
+        }
+
+        public override BindingElement Clone()
+        {
+            return new AzureQueueStorageTransportBindingElement();
+        }
```

Comment:
You should have a protected constructor which takes an instance of this class and pass this to it. In that constructor, you should copy values across. Basically like this:
```c#
protected AzureQueueStorageTransportBindingElement(AzureQueueStorageTransportBindingElement other) : base(other)
{
    MaxReceivedMessageSize = other.MaxReceivedMessageSize;
}

public override BindingElement Clone()
{
    return new AzureQueueStorageTransportBindingElement(this);
}
```

Reply:
Got it. updated.

Reply:
You didn't update the Clone method to use the new protected constructor.